cs2/src/state/player.rs

```diff
@@ -35,6 +35,7 @@ pub struct PlayerPawnInfo {
     pub controller_entity_id: Option<u32>,
     pub pawn_entity_id: u32,
     pub team_id: u8,
+    pub weapon_player_entity_id: u32,
```

Comment:
pawn_entity_id and weapon_player_entity_id should be equal, so this seems useless